active/NumberBox/NumberBox.md

```diff
@@ -57,17 +58,30 @@ example code with each description. The general format is:
 <!-- As an example of this section, see the Examples section for the PasswordBox control 
 (https://docs.microsoft.com/windows/uwp/design/controls-and-patterns/password-box#examples). -->
 
+<!-- Todo: Needs an input validation example, and probably a section in the Remarks about input validation. -->
+
 ### Create a simple NumberBox
 
 XAML
+
+<!-- todo: need a picture that matches the markup (the placeholder suggests a calculation, but this sample doesn't enable calculations-->
+
+<!-- todo: Can we remove these names? They don't serve any purpose here, and just add noise. And we should x:Bind the value, so that the example shows how you get the value out. -->
+
 ```XAML
-<NumberBox x:Name="EquationInputNumberBox" Header="Enter equation:" PlaceholderText="A + B * C" />
+<NumberBox x:Name="EquationInputNumberBox" Header="Enter equation:" />
 ```
-
 ![](images/numberbox-placeholder-text.png)
 
 ### Enable calculation support
 
+To let the user enter equations, use the `AcceptsCalculation` property. This default to false.
+
+<!-- todo: Need to briefly tell the story of calculations here, I'm not sure how it works. When the user hits enter, does it do the calculation? Does the user have to enter the '='? Does it keep showing the equation after it's done the calculation or does setting this property just allow me to type non-numerics into the box?
```

Comment:
Currently, when the user hits enter or the control loses focus, it calculates the value of the expression. = is not required (or allowed). The text input is replaced with the answer at that time.

Reply:
So neither OneNote nor Excel behavior. I think that's OK though; I think Excel behavior makes sense, but it could be compatibly be added later.

Reply:
What if there's a syntax error in the equation?

If you Alt+Tab away, does that count as lost focus, and so it evaluates the equation?

Reply:
Agreed. This was temporarily left empty while we were determining how evaluation is triggered. I will add some crisp documentation now that this has been defined.

@teaP, Mike raises an excellent point that NumberBox should accept a leading "=" as not throw Excel-minded users. Given time constraints, I can file this as a bug if you prefer?

Reply:
@MikeHillberg, errors will block evaluation and trigger a validation warning (unless the developer has opted out of our prescribed validation warning options).

Reply:
Sure, let's file a bug on the = thing. Currently the only validation we have is not changing the value to anything invalid, so if the user types in an invalid expression, the value just doesn't change and the text reverts back to displaying the current value.

Reply:
Will do,

And that is correct - right now we only have the "override invalid input" and "I'll take care of validation myself" options. Come 3.0, we will be able to add options to display validation warnings for the end user.

Reply:
I think it's better to leave out the '=' prefix now, and save it for later when it "works". I.e. when I do "=3+2" and tab away I get '5', but when I tab back I get "=3+2".

Reply:
@MikeHillberg, it is not an ask for the control to preserve history/for expressions to be accessible post evaluation. Does a leading "=" imply that functionality? If so, it should not be supported at this time.

Reply:
Yes, I think an interesting future feature would be to preserve the expression if it's prefixed with an '='. In the meantime, it would be better to not support the prefix at all, so that we're reserving the ability to add it in the future.

Reply:
Noted. I'll add it to the list of features to consider for V2 and resolve this conversation.